pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -118,6 +118,7 @@ func (r *CPUSuppress) calculateBESuppressCPU(node *corev1.Node, nodeMetric *metr
 	podMetrics []*metriccache.PodResourceMetric, podMetas []*statesinformer.PodMeta, beCPUUsedThreshold int64) *resource.Quantity {
 	// node, nodeMetric, podMetric should not be nil
 	nodeUsedCPU := &nodeMetric.CPUUsed.CPUUsed
+	metrics.RecordNodeUsedCPU(float64(nodeUsedCPU.Value()))
```

Comment:
How about recording the metric inside MetricAdvisor since it is a common metric? Otherwise, if we only concern with the collected node usage in the CPUSuppress feature, we'd better rename it as `be_suppress_node_used_cpu_cores`.

Reply:
It's a good idea to record `node_used_cpu_cores` metric by `metricsadvisor` package.  I will adjust it and squash the commits.